ci-operator/config/stackrox/scanner/stackrox-scanner-nightlies.yaml

```diff
@@ -0,0 +1,225 @@
+base_images:
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  ubi-minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  project_image:
+    dockerfile_path: .openshift-ci/Dockerfile.build_root
+images:
+- dockerfile_path: .openshift-ci/build/Dockerfile.generate-genesis-dump
+  from: src
+  to: genesis-dump
+- dockerfile_path: .openshift-ci/build/Dockerfile.generate-db-dump
+  from: genesis-dump
+  to: db-dump
+- dockerfile_path: .openshift-ci/build/Dockerfile.build-bundle
+  from: genesis-dump
+  to: scanner-bundle
+- dockerfile_path: .openshift-ci/build/Dockerfile.build-db-bundle
+  from: db-dump
+  to: scanner-db-bundle
+- dockerfile_path: Dockerfile
+  from: ubi-minimal
+  inputs:
+    scanner-bundle:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/stackrox/scanner/image/scanner/rhel/.
+  to: scanner
+- dockerfile_path: Dockerfile.slim
+  from: ubi-minimal
+  inputs:
+    scanner-bundle:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/stackrox/scanner/image/scanner/rhel/.
+  to: scanner-slim
+- dockerfile_path: Dockerfile
+  from: ubi
+  inputs:
+    scanner-db-bundle:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/stackrox/scanner/image/db/rhel/.
+  to: scanner-db
+- dockerfile_path: Dockerfile.slim
+  from: ubi
+  inputs:
+    scanner-db-bundle:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/stackrox/scanner/image/db/rhel/.
+  to: scanner-db-slim
+resources:
+  '*':
+    requests:
+      cpu: 2000m
+      memory: 4000Mi
+test_binary_build_commands: .openshift-ci/dispatch.sh noop
+tests:
+- as: merge-db-integration-tests
```

Comment:
I dropped the `merge-` in stackrox to make the names a little shorter and maybe easier to grasp when they go into TestGrid.